client/src/components/Form/FormMessage.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <b-alert :variant="variant" :show="showAlert">
+        {{ message }}
```

Comment:
If you use the standard mount, localization will be available either as a filter, or a directive.
```
{{ message | l }}
```

Reply:
Thanks. Done.